README.md

```diff
@@ -49,8 +49,8 @@ on Wikipedia.
 Installation
 ------------
 
-This software depends on [NumPy and Scipy], two Python packages for
-scientific computing. You must have them installed prior to installing
+This software depends on [NumPy], a Python package for
+scientific computing. You must have it installed prior to installing
```

Comment:
Does it really have to be installed prior to installing `gensim` having in mind that a proper _numpy_ version is already specified in _build-system.requires_ table? Meaning it will be downloaded automatically during _gensim_ build phase.

If it does not --- what should we do with regard to the next paragraph about BLAS lib? I think it will not be customisable for any architecture that a _numpy_ .whl build exists for.

Reply:
Yes, I think the sentence starting "You must" is incorrect. Numpy will get installed automatically if it already isn't there.

With BLAS, I think we just leave that note as-is. You can't customize it as part of the gensim install. If you want to mess with BLAS, then you have to do it yourself, _before_ installing numpy and gensim.

Right @piskvorky ?

Reply:
I forgot how exactly it worked back then. IIRC numpy was needed for the installation (numpy actively used during install, down to its C API). Maybe scipy too. I believe that is the intent behind *"You must have [numpy] installed prior to installing gensim"*.

If `pip` installs numpy automatically before the gensim install itself starts, that's great. Some people don't use `pip`, I don't know all the deployment methods people use (from source, conda, etc) and whether they all install dependencies first.

In any case that was 15 years ago, the current ecosystem might work differently, and none of this is needed.

Reply:
As I see it, the problem of linking Numpy to any BLAS lib is a matter to worry about only during building Numpy from source. 
This can occur only if: 
  * one wants to customise his Numpy build in a non typical way. In this case I believe such person knows what he's doing so this note is not really valuable for him, 

  * one installs gensim which in turn downloads appropriate NumPy build (but src distribution instead of a .whl). Then the pip (or I believe any package manager) would try building the NumPy from source and implicitly linking an existing BLAS lib. 
  
What's more, a NumPy installed as a gensim's dependency will not be (in some cases) the same NumPy as used for building gensim. Pip by default builds a library in a separate temporary virtual env where it installs build-only dependencies.

I've redacted the paragraph. Please review whether it sounds sensible .